include/rocksdb/rate_limiter.h

```diff
@@ -91,7 +91,11 @@ class RateLimiter {
 
   // Total # of requests that are pending for bytes in rate limiter
```

Comment:
We need more description in here like

    For convenience, this function is implemented by the RateLimiter returned by NewGenericRateLimiter but is not required by RocksDB. The default implementation indicates "not supported."